### ISSUE TITLE
request: think about ways that we could incorporate a feed that shows all the posts of people you follow

### ISSUE THREAD

**Issue by shea256** [jay-ithiel/guild#21]
I'm thinking that we'd need to add a social/following element to the Blockstack Browser to enable this.

We have plans to do this already and actually we have this hidden feature to follow people on onename.com.

When you sign in with Blockstack, your followers and followees could come with you and your feed could be updated every time someone posts a new story and every time you follow someone new. Your social graph on Guild is one and the same with the social graph of Blockstack identities.

Unfortunately this will involve quite a bit of complexity so my guess is we'll have to have a few discussions about this and think about how the feed will be loaded and such.

**Comment by jay-ithiel** [comment]
Yeah I definitely would like to add a follow feature with a feed. Let's do it
